src/components/kpi/kpi.directive.coffee

```diff
@@ -98,11 +111,17 @@ angular
         $scope.hasValidTargets = ->
           ImpacKpisSvc.validateKpiTargets($scope.targets)
 
-        $scope.updateSettings = ->
+        # Prevent displaying text characters before expresses are loaded.
+        $scope.buildEphasis = (kpi)->
```

Comment:
I get why you're doing that here, but not a big fan: this belongs to the view...
Maybe simply wrap it into a `div` or `span` with a `ng-if` condition?